packages/@aws-cdk/cloudformation-include/test/test-templates/fn-sub-parameters.json

```diff
@@ -0,0 +1,22 @@
+{
+  "Parameters": {
+    "MyParam": {
+      "Type": "String"
+    }
+  },
+  "Resources": {
+    "Bucket": {
+      "Type": "AWS::S3::Bucket",
+      "Properties": {
+        "BucketName": { 
+          "Fn::Sub": [ 
+            "${MyParameter}-${MyParam}", 
+            { 
+              "MyParameter": { "Ref" : "MyParam" }
```

Comment:
Aren't we checking this already in `fn-sub-parameter-shadow.json `? Shouldn't this be just:

```json
{
  "Parameters": {
    "MyParam": {
      "Type": "String"
    }
  },
  "Resources": {
    "Bucket": {
      "Type": "AWS::S3::Bucket",
      "Properties": {
        "BucketName": { 
          "Fn::Sub": "${MyParam}"
        }
     }
   }
}
```